packages/desktop-client/playwright.config.js

```diff
@@ -32,6 +32,15 @@ expect.extend({
       return darkmode;
     }
 
+    // Switch to darkmode and check
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
    // Switch to midnight theme and check
```